environment.md

```diff
@@ -1,19 +1,35 @@
-# Environment
+# Server Environment
 
-WordPress will run in a minimum server environment. However, WordPress doesn't run optimally on the minimum system requirements. This section will cover the _recommended_ server environment for WordPress to run more effectively.
+Although WordPress can work in almost any environment, even very minimal ones, it must be acknowledged that it does not work completely well in these. That's why here we are going to make some minimum recommendations of the environment in which it would work most effectively.
 
-## Recommended Servers
+## Web Server
 
-WordPress recommends using one of these servers:
+The web server is the system where the files of the web site are hosted and where the users arrive to consult them. There are many web servers, and in principle anyone who supports a PHP connection should be able to work with WordPress.
 
-*   Apache 2.4 or higher
-*   NGINX 1.14 or higher
+When it comes to the server, web, WordPress works best with these (sorted alphabetically):
+
+* [Apache HTTPD](https://httpd.apache.org/) 2.4.x
```

Comment:
The versions are based on "secure and maintained" versions... so, as we talked before, we should only recommend "stable versions".

Reply:
Added a text like:

_WordPress may work with older versions, but we recommend only versions that are stable and supported._

Reply:
I don't think we should make recommendations for servers that are not recommended by WordPress. If we want to change that recommendation, it'd be a good idea to chat with the folks who decided on those servers, since there's a likely reason they are not included.

Reply:
The question is... why WordPress recommends only Apache / nginx and not others? (who decides that?) I know some hosters using LiteSpeed and OpenLiteSpeed.

I don't have any problem to only leave what there is, but I think our handbook is "for hosters" and not "WordPress Requirements" so, we should give light to hosters (also, I think we should add more web servers ;) if they work or hosters have working).

But, we should try and discuss that on the Meeting to know what people are using and then, do a Requirements recommendation?

Reply:
Yes! I think that we need to find out why it is not supported, and see how we should talk about it. We can ask around a bit. I suspect either the core or meta team will know the most.

This documentation is official WordPress documentation. The handbook is official WordPress hosting recommendations, and this page specifically is linked from the requirements page: https://wordpress.org/about/requirements/